src/main/java/com/limechain/runtime/RuntimeStorageKey.java

```diff
@@ -0,0 +1,14 @@
+package com.limechain.runtime;
+
+import com.limechain.trie.structure.nibble.Nibbles;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+// all available keys can be found here: https://www.shawntabrizi.com/substrate-known-keys/
+@Getter
+@AllArgsConstructor
+public enum RuntimeStorageKey {
+    GENESIS_SLOT(Nibbles.fromHexString("0x1cb6f36e027abb2091cfb5110ab5087f678711d15ebbceba5cd0cea158e6675a"));
```

Comment:
```suggestion
    GENESIS_SLOT("babe genesisSlot");
```

We can have the keys in their string form and have a method hex and return nibbles.

Reply:
Keeping the keys in human-readable form means that we should then encode them with the xxHash algorithm, which we don't have as a dependency at the moment, so I decided to use the encoded form directly.

Reply:
ok